utils.py

```diff
@@ -279,17 +280,31 @@ def save_bitmasked_ROIs(context, labels, file_input, data, affine):
         file_input (flywheel.models.file.File): Input File object
         data (numpy.ndarray): The nifti data object
         affine (numpy.ndarray): The nifti affine
+        combined_output_size (string): numpy integer size
     """
 
-    if len(labels) > 63:
+    if combined_output_size == 'int8':
```

Comment:
This is good!
This is well thought out and implemented.

My only concern here is that, should they experiment with multiple `combined_output_size` settings, they will may not see a significant decrease in file size due to the NIfTI compression. 

However, I do think it is good to give them this level of control!!!

Reply:
You're completely right!  My intent was not to decrease the size but deliver file formats that are easy to read in.  int64 would break my software -- I don't think that option was available when I started coding my manual labeling stuff.  Even if it was, I never would have thought it was necessary.  

I thought saving in different sizes would be a quick modification and I was almost right.  Things always take longer than I expect.